client/App/Components/AudioView.js

```diff
@@ -0,0 +1,252 @@
+var React = require('react-native');
+var MapView = require('react-native-maps');
+var Icon = require('react-native-vector-icons/FontAwesome');
+var CircleMarker = require('./CircleMarker');
+var PhotoView = require('./PhotoView');
+var PhotosView = require('./PhotosView');
+var api = require('../Utils/api');
+var BlackPhotoMarker = require('./BlackPhotoMarker');
+var RedPhotoMarker = require('./RedPhotoMarker');
+var IconIon = require('react-native-vector-icons/Ionicons');
+
+var {AudioRecorder, AudioPlayer} = require('react-native-audio');
+
+var {
+  Navigator,
+  StyleSheet,
+  View,
+  Text,
+  Dimensions,
+  TouchableOpacity,
+  TouchableHighlight,
+  StatusBarIOS,
+  ActivityIndicatorIOS,
+  TextInput,
+  Component,
+  DeviceEventEmitter,
+  AlertIOS,
+  AppRegistry
+} = React;
+
+class AudioView extends React.Component {
```

Comment:
Is it possible to call it MicView? AudioView might need to be used later when playing audio files. ya dig?